### PR TITLE
feat: differentiate ok actions and update alarm thresholds

### PR DESCRIPTION
**Pull request by mohamed-cds**
This PR makes it easier to distinguish ok actions and alarms. Production thresholds for the newly introduced inflight and bulk buffer metrics are being increased to exceed `normal` traffic

### CODE REVIEW

File: aws/common/cloudwatch_alarms.tf [hunk 1]
```diff
@@ -741,7 +741,7 @@ resource "aws_cloudwatch_metric_alarm" "inflights-not-being-processed-critical"
 }
 
 resource "aws_cloudwatch_metric_alarm" "bulk-not-being-processed-warning" {
-  alarm_name          = "bulk-not-being-processed-warning"
+  alarm_name          = "bulk-buffer-not-being-processed-warning"
```
Comment (jimleroyer) [inline aws/common/cloudwatch_alarms.tf:744]: Thanks!

File: env/production/common/terragrunt.hcl [hunk 2]
```diff
@@ -12,8 +12,8 @@ inputs = {
...
   sns_monthly_spend_limit_us_west_2                         = 2000
   alarm_warning_document_download_bucket_size_gb            = 100
-  alarm_warning_inflight_processed_created_delta_threshold  = 50
-  alarm_critical_inflight_processed_created_delta_threshold = 100
-  alarm_warning_bulk_processed_created_delta_threshold      = 50
-  alarm_critical_bulk_processed_created_delta_threshold     = 100
+  alarm_warning_inflight_processed_created_delta_threshold  = 100
+  alarm_critical_inflight_processed_created_delta_threshold = 200
+  alarm_warning_bulk_processed_created_delta_threshold      = 5000
+  alarm_critical_bulk_processed_created_delta_threshold     = 10000
```
Comment (jimleroyer) [inline env/production/common/terragrunt.hcl:18]: That escalated quickly. 😅

File: aws/common/sns.tf [hunk 3]
```diff
@@ -119,3 +157,15 @@ resource "aws_sns_topic_subscription" "alert_critical_us_west_2_to_opsgenie" {
   raw_message_delivery   = false
   endpoint_auto_confirms = true
 }
+
+resource "aws_sns_topic_subscription" "alert_critical_us_west_2_to_opsgenie_ok" {
+  provider = aws.us-west-2
+
+  count = var.env == "production" ? 1 : 0
```
Comment (jimleroyer) [inline aws/common/sns.tf:164]: How is the count used here?

**Review by jimleroyer — APPROVED** [review]
LGTM

File: aws/common/sns.tf (hunk 3 above)
Comment (mohamed-cds) [reply aws/common/sns.tf:164]: @jimleroyer it checks whether the environment is `production` if it's false it'll set the count to 0 which will result in the resource not being created. It's a common way in TF to only deploy certain resources to prod vs staging, etc.

**Pull request merged**